casepro/cases/tests.py

```diff
@@ -291,6 +292,36 @@ def test_get_all(self):
         self.assertEqual(set(Case.get_closed(self.unicef)), {case2})
         self.assertEqual(set(Case.get_closed(self.unicef, user=self.user1, label=self.pregnancy)), {case2})
 
+    def test_get_all_passed_response_time(self):
+        bob = self.create_contact(self.unicef, 'C-002', "Bob")
+        cat = self.create_contact(self.unicef, 'C-003', "Cat")
+        nic = self.create_contact(self.nyaruka, 'C-104', "Nic")
```

Comment:
We probably also don't need nic now that the extra messages are gone (sorry nic)